docs/basics/101-135-help.rst

```diff
@@ -118,40 +118,124 @@ The "submit a question link" on `DataLad's GitHub page <https://github.com/datal
 page prefills a neurostars form with a template for a question for a good
 starting point if you want to have more guidance or encounter writer's block.
 
+Common warnings and errors
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Common git-annex warnings and errors
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A lot of output you will see while working with DataLad originates from warnings
+or errors by DataLad, git-annex, or Git.
+Some of these outputs can be wordy and not trivial to comprehend - and even if
+everything works, some warnings can be hard to understand.
+This following section will list some common git-annex warnings and errors and
+attempts to explain them. If you encounter warnings or errors that you would
+like to see explained in this book, please let us know by
+`filing an issue <https://github.com/datalad-handbook/book/issues/new>`_.
 
-A lot of output you will see while working with DataLad originates from
-git-annex. It's outputs can be wordy and not trivial to comprehend even if
-everything works. This following section will list some common git-annex
-warnings and errors and attempts to explain them.
+Output produced by Git
+""""""""""""""""""""""
+
+**Unset Git identity**
+
+If you have not configured your Git identity, you will
+see warnings like this when running any DataLad command::
+
+   [WARNING] It is highly recommended to configure git first (set both user.name and user.email) before using DataLad.
+
+To set your Git identity, go back to section :ref:`installconfig`.
+
+
+**Rejected pushes**
+
+One error you can run into when publishing dataset contents is that your
+:command:`datalad push` to a sibling is rejected.
+One example is this::
+
+   $ datalad push --to public
+    [ERROR  ] refs/heads/master->public:refs/heads/master [rejected] (non-fast-forward) [publish(/home/me/dl-101/DataLad-101)]
+
+This example is an attempt to push a local dataset to its sibling on GitHub. The
+push is rejected because it is a ``non-fast-forward`` merge situation. Usually,
+this means that the sibling contains changes that your local dataset does not yet
+know about. It can be fixed by updating from the sibling first with a
+:command:`datalad update --merge`.
+
```

Comment:
We could try to make the summary result renderer of ˋpushˋ give this hint.

Reply:
Good idea!